### PR TITLE
fix: Data Proxy docs update

### PR DESCRIPTION
**Pull request by ghost**
## Describe this PR
Minor fixes/updates to the docs for Data Proxy

### CODE REVIEW

**Review by albertoperdomo — APPROVED** [review]
LGTM. Capitalization is something @tanberry was looking into so I will leave that up to her and also for her to review the wording specifically.

I would wait for her review to merge.

File: content/200-concepts/100-components/07-prisma-data-platform.mdx
Comment (tanberry) [inline content/200-concepts/100-components/07-prisma-data-platform.mdx:63]: Let's change to" ### Why use the Prisma Data Proxy?"

**Review by tanberry — APPROVED** [review]

**Pull request merged**
